javascript/script.js

```diff
@@ -42,23 +42,43 @@ const addToDoItem = (toDoItem) => {
 			emptyImage.remove();
 			$("list-items").classList.remove("add-empty-image");
 			$("search-text").classList.remove("hidden");
+			$("image-inside-searchbar").classList.remove("hidden");
 		}
 	}
-	let newItem = "<li id='item-" + toDoList[toDoList.length - 1].id + "' ><div class='list-content'>" +
-		"<input " + (toDoList[toDoList.length - 1].taskCompleted ? "checked" : "") + " type='checkbox' class='checkbox' id='check-" + toDoList[toDoList.length - 1].id + "'>" +
-		"<span class='descriptionAndDate'><span id = 'desc-" + toDoList[toDoList.length - 1].id + "' class='description " + (toDoList[toDoList.length - 1].taskCompleted ? "strike-through" : "") + "'>" + toDoList[toDoList.length - 1].description + "</span>" +
-		"<div class='descriptionDate'>" + toDoList[toDoList.length - 1].date + "</div></span>" +
-		"</div>" +
-		"<div class='flex'>" +
-		"<button id='mobile-save-" + toDoList[toDoList.length - 1].id + "' class='fa-light fa fa-floppy-o mobile-save-button hidden'></button>" +
-		"<a id='menu-" + toDoList[toDoList.length - 1].id + "' class='fa-dark fa fa-ellipsis-v menu-button'></a>" +
-		"</div>" +
-		"<div class='buttons' id='buttons'>" +
-		"<button id='edit-or-save-" + toDoList[toDoList.length - 1].id + "' class='edit-button fa-light fa fa-pencil'></button>" +
-		"<button id='remove-" + toDoList[toDoList.length - 1].id + "' class='remove-button fa-light fa fa-trash-o'></button>" +
-		"</div></li>";
-
-	$("printing-list").insertAdjacentHTML('afterbegin', newItem);
+	let newItem = $('new-item-template').content.cloneNode(true)
+	let item = newItem.getElementById("item");
+	item.id = 'item-' + toDoList[toDoList.length - 1].id;
+
+	let check = newItem.getElementById("check");
+	if (toDoList[toDoList.length - 1].taskCompleted) {
+		check.setAttribute('checked', 'true');
+	}
+	check.id = 'check-' + toDoList[toDoList.length - 1].id;
+
+	let desc = newItem.getElementById("desc");
+	desc.innerHTML = toDoList[toDoList.length - 1].description;
```

Comment:
Setting `innerHTML` is unnecessary here. `description` is not HTML content. Use `innerText` instead.